### PR TITLE
Docs: Write the Quick start CDN guide.

### PR DESCRIPTION
**Pull request by gorzelinski**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: Write the Quick start CDN guide. Closes https://github.com/cksource/ckeditor5-commercial/issues/5975.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

**Review by godai78 — APPROVED** [review]

File: docs/getting-started/quick-start.md [hunk 1]
```diff
@@ -72,15 +67,107 @@ import { FormatPainter, SlashCommand } from 'ckeditor5-premium-features';
...
+```
+
+Then, you need to attach the script with the JavaScript code. To simplify imports, you can use the feature available in browsers &ndash; the [import map](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type/importmap). It maps the text used as the module specifier and the value that will replace the text when resolving the specifier. This way, import statements do not need to contain the full CDN URL.
+
+```html
+<script type="importmap">
+	{
+		"imports": {
+			"ckeditor5": "<CDN_LINK>/ckeditor5/dist/index.min.js",
+			"ckeditor5/": "<CDN_LINK>/ckeditor5/"
```
Comment (Witoso) [inline docs/getting-started/quick-start.md:103]: @filipsobol is this always needed?

**Review by Witoso — COMMENTED** [review]
LGTM.

File: docs/getting-started/quick-start.md [hunk 2]
```diff
@@ -25,38 +25,33 @@ npm install ckeditor5
 
 Now you can import all the modules from the `ckeditor5` package. Additionally, you have to import CSS styles separately.
 
-**Importing and registering UI translations is optional for the English language.** For any other language, use imported translations, as in the example below.
+**Importing and registering UI translations is optional for the English language.** For any other language, use imported translations, as in the {@link getting-started/setup/configuration setup section}.
```
Comment (filipsobol) [inline docs/getting-started/quick-start.md:28]:
```suggestion
**Importing and registering UI translations is optional for the American English.** To use the editor in any other language, import translations as shown in the {@link getting-started/setup/configuration setup section}.
```

File: docs/getting-started/quick-start.md [hunk 3]
```diff
@@ -72,15 +67,107 @@ import { FormatPainter, SlashCommand } from 'ckeditor5-premium-features';
...
 		licenseKey: 'your-license-key'
 	} )
-	.catch( err => {
-		console.error( err );
-	} );
+```
+
+## Using CKEditor&nbsp;5 from CDN
+
+CDN is an effortless method of running CKEditor 5. You can start using it in just a few steps and with a few tags.
```
Comment (filipsobol) [inline docs/getting-started/quick-start.md:80]: I'd be cautious about using the word `effortless`, as - depending on the setup - it may require more work than the NPM setup. I'd suggest replacing `effortless` with `alternative`.

Comment (filipsobol) [inline docs/getting-started/quick-start.md:80]: I'd be cautious about using the word `effortless`, as - depending on the setup - it may require more work than the NPM setup. I'd suggest replacing `effortless` with `alternative`.

```suggestion
CDN is an alternative method of running CKEditor 5. You can start using it in just a few steps.
```

File: docs/getting-started/quick-start.md [hunk 4]
```diff
@@ -72,15 +67,107 @@ import { FormatPainter, SlashCommand } from 'ckeditor5-premium-features';
...
+
+### Obtain a license key
+
+To activate CKEditor&nbsp;5 premium features, you will need a commercial license. If you wish to purchase a commercial CKEditor&nbsp;5 license or a license to one of the premium features, [contact us](https://ckeditor.com/contact/?sales=true#contact-form) to receive an offer tailored to your needs. To obtain an activation key, please follow the {@link support/license-key-and-activation License key and activation} giuide.
```
Comment (filipsobol) [inline docs/getting-started/quick-start.md:84]:
```suggestion
To activate CKEditor&nbsp;5 premium features, you will need a commercial license. If you wish to purchase a commercial CKEditor&nbsp;5 license or a license to one of the premium features, [contact us](https://ckeditor.com/contact/?sales=true#contact-form) to receive an offer tailored to your needs. To obtain an activation key, please follow the {@link support/license-key-and-activation License key and activation} guide.
```

File: docs/getting-started/quick-start.md [hunk 5]
```diff
@@ -72,15 +67,107 @@ import { FormatPainter, SlashCommand } from 'ckeditor5-premium-features';
...
+
+<!-- Not sure we should duplicate this in QS: TODO: Describe the steps to obtain a license key, attaching some screenshots. -->
+
+### CDN setup
+
+Start by attaching a link to the style sheet. It contains all styles for the editor's UI and content. You can also include your styles if you like. Refer to the content styles guide for more information.
```
Comment (filipsobol) [inline docs/getting-started/quick-start.md:90]: Add link to the `content styles` guide.

File: docs/getting-started/quick-start.md [hunk 6]
```diff
@@ -72,15 +67,107 @@ import { FormatPainter, SlashCommand } from 'ckeditor5-premium-features';
...
+
+```html
+<link rel="stylesheet" href="<CDN_LINK>/ckeditor5/dist/styles.css" />
+```
+
+Then, you need to attach the script with the JavaScript code. To simplify imports, you can use the feature available in browsers &ndash; the [import map](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type/importmap). It maps the text used as the module specifier and the value that will replace the text when resolving the specifier. This way, import statements do not need to contain the full CDN URL.
```
Comment (filipsobol) [inline docs/getting-started/quick-start.md:96]: This feature is not just used to make imports look nicer, so I would rephrase that so as not to tempt anyone to skip using it.

```suggestion
Next, we need to add an [import map](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type/importmap) that allows us to map an easy-to-remember specifier (like `ckeditor5`) to the full URL of the file from the CDN. We use this browser feature to share an editor engine code between plugins.
```

File: docs/getting-started/quick-start.md (hunk 1 above)
Comment (filipsobol) [reply docs/getting-started/quick-start.md:103]: It's only needed when importings translations, but I'm fine with having this here.

File: docs/getting-started/quick-start.md [hunk 7]
```diff
@@ -72,15 +67,107 @@ import { FormatPainter, SlashCommand } from 'ckeditor5-premium-features';
...
+	{
+		"imports": {
+			"ckeditor5": "<CDN_LINK>/ckeditor5/dist/index.min.js",
+			"ckeditor5/": "<CDN_LINK>/ckeditor5/"
+		}
+	}
+</script>
+```
+
+After you  set up the import map, you can add plugins. In the following script tag, import the desired plugins and add them to the plugins array. Notice that both script tags have the corresponding `type` values.
```
Comment (filipsobol) [inline docs/getting-started/quick-start.md:109]:
```suggestion
Once you've added the import map, you can access editor and its plugins using the `ckeditor5` specifier. In the following script tag, we import the desired plugins and add them to the `plugin' array. Note that this and the previous script tags have the appropriate `type` values.
```

File: docs/getting-started/quick-start.md (hunk 6 above)
Comment (godai78) [reply docs/getting-started/quick-start.md:96]: I'd suggest sticking to "you" as this is the dominant across the whole guide.

File: docs/getting-started/quick-start.md (hunk 7 above)
Comment (godai78) [reply docs/getting-started/quick-start.md:109]: No contractions.

File: docs/getting-started/quick-start.md (hunk 2 above)
Comment (godai78) [reply docs/getting-started/quick-start.md:28]: Unless we have separate translations for American and British (if so, we should reflect it in the docs immediately), this change makes very little sense.

File: docs/getting-started/quick-start.md (hunk 3 above)
Comment (gorzelinski) [reply docs/getting-started/quick-start.md:80]: Yeah, you're right. That was marketing jabber foremost. Will do.

File: docs/getting-started/quick-start.md (hunk 5 above)
Comment (gorzelinski) [reply docs/getting-started/quick-start.md:90]: Sure, but we don't have this guide in the new structure yet.

File: docs/getting-started/quick-start.md (hunk 6 above)
Comment (gorzelinski) [reply docs/getting-started/quick-start.md:96]: Good point. But I would leave "we" in the last sentence because there is a rationale for why we recommend this feature.

Comment (godai78) [reply docs/getting-started/quick-start.md:96]: Sure.

**Comment by gorzelinski** [comment]
To consider: if we want to add steps to obtain a license key. On the one hand, the guide will cover all steps. On the other, we duplicate content.

File: docs/getting-started/quick-start.md (hunk 2 above)
Comment (filipsobol) [reply docs/getting-started/quick-start.md:28]: As far as I can see, we have `en` (default), `en-au` and `en-gb`. That's why I suggested explicitly saying that **American English** is the default.

**Comment by gorzelinski** [comment]
To do: add missing links when they are available.

File: docs/getting-started/quick-start.md [hunk 8]
```diff
@@ -25,7 +25,7 @@ npm install ckeditor5
 
 Now you can import all the modules from the `ckeditor5` package. Additionally, you have to import CSS styles separately.
 
-**Importing and registering UI translations is optional for the English language.** For any other language, use imported translations, as in the {@link getting-started/setup/configuration setup section}.
+**Importing and registering UI translations is optional for American English.** To use the editor in any other language, use imported translations, as shown in the {@link getting-started/setup/configuration setup section}.
```
Comment (godai78) [inline docs/getting-started/quick-start.md:28]:
```suggestion
**Importing and registering UI translations is optional for English.** To use the editor in any other language, use imported translations, as shown in the {@link getting-started/setup/configuration setup section}.
```

Comment (godai78) [reply docs/getting-started/quick-start.md:28]: I insist. We do not have separate translations, do we?

Comment (gorzelinski) [reply docs/getting-started/quick-start.md:28]: It seems that we have:

> As far as I can see, we have en (default), en-au and en-gb. That's why I suggested explicitly saying that American English is the default.

I also asked @pszczesniak, and he confirmed.

Comment (godai78) [reply docs/getting-started/quick-start.md:28]: Why don't we have this documented and only list English in out langauge guide? o\_O This is a pretty meaningful rewrite.  
  
OK\< Keep American English, then. I'll split the issue on that.

Comment (gorzelinski) [reply docs/getting-started/quick-start.md:28]: I looked through the language guides, and you're right - there is only "English" without dialects.

Comment (godai78) [reply docs/getting-started/quick-start.md:28]: We actually do have community translations for these two bad boys: \`au\` and `gb` https://github.com/ckeditor/ckeditor5/tree/master/packages/ckeditor5-core/lang/translations:
